### PR TITLE
Modify universal-robots-client-library build to export more files

### PR DESCRIPTION
**Pull request by FilipposSot**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @b-bremer, modules you maintain (universal-robots-client-library) have been updated in this PR. Please review the changes.

**Comment by FilipposSot** [comment]
@bazel-io skip_check unstable_url

File: modules/universal-robots-client-library/1.5.0.bcr.1/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
++    default_visibility = ["//visibility:public"],
++)
++
++exports_files([
++    "LICENSE_apache_2_0",
++    "include/ur_client_library/queue/LICENSE.md",
++    "include/ur_client_library/queue/atomicops.h",
++    "examples/resources/rtde_input_recipe.txt",
++    "examples/resources/rtde_output_recipe.txt",
++    "resources/external_control.urscript",
```
Comment (b-bremer) [inline modules/universal-robots-client-library/1.5.0.bcr.1/patches/add_build_file.patch:20]: AFAIK this is the only change

**Review by b-bremer — DISMISSED** [review]
Only change is adding 

```
+    "examples/resources/rtde_input_recipe.txt",
+    "examples/resources/rtde_output_recipe.txt",
+    "resources/external_control.urscript",
```

to the `exports_files` rule

**Review by bazel-io — DISMISSED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Review by b-bremer — APPROVED** [review]

File: modules/universal-robots-client-library/1.5.0.bcr.1/patches/add_build_file.patch (hunk 1 above)
Comment (FilipposSot) [reply modules/universal-robots-client-library/1.5.0.bcr.1/patches/add_build_file.patch:20]: Yes, thanks.

**Review by b-bremer — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**